main.go

```diff
@@ -295,6 +296,29 @@ func createBody(body string) io.Reader {
 	return strings.NewReader(body)
 }
 
+// getFilenameFromHeader tries to automatically determine the output filename,
+// when saving to disk, based on the Content-Disposition header.
+// If the header is not present, or it does not contain enough information to
+// determine which filename to use, this function returns "".
+func getFilenameFromHeader(header http.Header) string {
```

Comment:
Haha. Funny you mention it, I started off with `headers` and then switched because of the `net/http` type. 😂 

I'll happily revert that back to the plural form.